cms/templates/widgets/header.html

```diff
@@ -230,8 +230,15 @@ <h3 class="title"><span class="label"><a href="${get_online_help_info(online_hel
 
     % else:
       <%
-        login_url = settings.LMS_ROOT_URL + '/login'
         register_url = settings.LMS_ROOT_URL + '/register'
+
+        if settings.FEATURES.get('DISABLE_STUDIO_SSO_OVER_LMS', False):
+          login_url = settings.LOGIN_URL
+        else:
+          login_url = '{lms_root_url}/login?next={next_url}'.format(
+            lms_root_url=settings.LMS_ROOT_URL,
+            next_url=current_url,
+          )
```

Comment:
Can we consolidate the use of `DISABLE_STUDIO_SSO_OVER_LMS` to only the settings file?  So it's logic isn't spread throughout these other .html files?   As a concrete suggestion:

1. Add new settings in production.py for `FRONTEND_LOGIN_URL` and `FRONTEND_LOGOUT_URL`.
2. When `DISABLE_STUDIO_SSO_OVER_LMS` is `True`
    * Set `FRONTEND_LOGIN_URL` to `LOGIN_URL`
    * Set `FRONTEND_LOGOUT_URL` to `EDX_ROOT_URL + '/signout'`

3. When `DISABLE_STUDIO_SSO_OVER_LMS` is `False`
    * Set `FRONTEND_LOGIN_URL` to `LMS_ROOT_URL` + '/login'`
    * Set `FRONTEND_LOGOUT_URL` to LMS_ROOT_URL` + '/logout'`

4. Update these `.html` files to always append `?next={next_url}` regardless of the `DISABLE_STUDIO_SSO_OVER_LMS` value.  I believe the old Studio endpoints will just ignore the `next` parameter.